index.html

```diff
@@ -9,11 +9,72 @@
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
-<body>
+<header>
+    
+    <nav><img src="img/karma-logo.svg" width="17" height="17" style="margin-left:-5%">
+
+        <a href="#">Meet Karma</a>
+        <a href="#">How it Works</a>
+        <a href="#">Store</a>
+        <a href="#">Blog</a>
+        <a href="#">Help</a>
+        <a href="#">Login</a>
+    </nav>
+
+</header>
+<body class="background_img">
+
+    <header class="intro">
+
+    <h1>Introducing Karma</h1>
+    <p>Bring WIFI with you, everywhere you go.</p>
+    </header><br>
+
+    <a href="#">
+        <button class="button">Learn More</button>
+        </a><br>
+        <header>
+            <h2 class="h2">Everyone needs a little Karma</h2>
+            </header>
+            <header class="container">
+
+                <article class="article">
```

Comment:
The article is an element by himself. You don't need to use a class with the same name.

Reply:
Hey Catalina, 

Yea, i was actually playing around 
with some code and i forgot to remove them, otherwise the code was redundant.

Thanks again